portfolio/src/main/webapp/index.html

```diff
@@ -20,22 +20,29 @@ <h2>A bit about me</h2>
     <ul>Guilty pleasure: binge-watch The Big Bang Theory</ul>
     <ul>Fun fact: ❤raw carrots ⛔cooked carrots</ul>
     <br>
-    <h2>Where am I?</h2>
-    <button onclick="findZoe()">Find Zoe</button>
+    <h2>Where am I? <button onclick="findZoe()">Find Zoe</button></h2>
     <div id="link-container">
       <a id="link"></a>
     </div>
     <br>
-    <h2>Help me <button onclick="grow()"><b>Grow</b></button></h2>
+    <h2>Help me <button onclick="grow()">Grow</button></h2>
     <img id="basil" />
     <p id="endMsg"></p>
     <br>
     <h2>Comments</h2>
     <form action="/data" method="POST">
-      <p>Add a new comment: </p>
+      <label>Leave a comment </label>
       <input type="text" id="new-comment" name="new-comment">
-      <input type="submit" />
+      <input type="submit">
     </form>
+    <br>
+    <label>Display the latest comments </label>
+    <select name="limit" id="limit">
+      <option value="3" selected="selected">3</option>
+      <option value="10">10</option>
+      <option value="20">20</option>
+    </select>
+    <button type="button" onclick="getComments()">Refresh</button>    
```

Comment:
Maybe have select and button inside the same self contained element tag so it easier to read that the button will only use data that are in its siblings tags.

Reply:
It can be under a form element without an action or method attribute.